pkg/reconciler/v1alpha1/autoscaling/kpa/kpa.go

```diff
@@ -45,22 +45,22 @@ const (
 	controllerAgentName = "kpa-class-podautoscaler-controller"
 )
 
-// KPAMetrics is an interface for notifying the presence or absence of KPAs.
-type KPAMetrics interface {
-	// Get accesses the Metric resource for this key, returning any errors.
-	Get(ctx context.Context, namespace, name string) (*autoscaler.Metric, error)
+// KPADeciders is an interface for notifying the presence or absence of KPAs.
+type KPADeciders interface {
```

Comment:
I actually agree. I thought of going through those at some point, but since you're already making the change. Package name explicitly tells that it's KPA Decider.